src/Logging/PsrLoggerTrait.php

```diff
@@ -34,6 +34,16 @@
     protected abstract function getLogger();
 
     /**
+     * Return common labels for each log entry.
+     *
+     * @return array
+     */
+    protected function getLabels()
+    {
+        return [];
```

Comment:
Would it make sense to have the metadata provider live on `Logger`? With that approach, even if someone decides to write their own psr implementation (or they just use the `Logger` and don't need/care about PSR-3) the appropriate labels would still carry through.